packages/pieces/http/src/lib/actions/send-http-request-action.ts

```diff
@@ -24,10 +30,19 @@ export const httpSendRequestAction = createAction({
       displayName: 'Body',
       required: false,
     }),
+    failsafe: Property.Checkbox({
+      displayName: "On error don't abort",
```

Comment:
I am suggesting to change display Name to be something like this:

`No Error on Failure`

What do you think?

Reply:
Works. Will do the necessary change.